src/Tests/Microsoft.NET.TestFramework/Utilities/CommonExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.NET.TestFramework.Utilities
+{
+    public static class CommonExtensions
```

Comment:
Why did you move these methods out of DirectoryInfoExtensions and FileInfoExtensions?  I would just leave them where they are unless there's a reason to move them.

Reply:
It makes sense that they should be in `Utilities` namespace.